runtime/common/src/lib.rs

```diff
@@ -1330,6 +1358,69 @@ macro_rules! create_runtime_api {
                 }
             }
 
+            impl pallet_contracts::ContractsApi<Block, AccountId, Balance, BlockNumber, Hash>
```

Comment:
The formatting seems off in a couple of places here (Allman brackets and missing trailing comma).

Reply:
I put that code into a file inside the Zeitgeist repository and executed `cargo fmt`. In regards to the Allman brackets, that's what cargo fmt does. In case of the trailing comma, https://github.com/zeitgeistpm/zeitgeist/pull/1012/commits/17a6ef86cd4b822d3576a277c1038513d333a7c3.